api/LsLink.py

```diff
@@ -0,0 +1,217 @@
+
+"""
+    Copyright (c) 2015-2016 Cisco Systems, Inc. and others.  All rights reserved.
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v1.0 which accompanies this distribution,
+    and is available at http:#www.eclipse.org/legal/epl-v10.html
+"""
+from Base import *
+from FieldProcessors import *
+
+class LsLink(Base):
+    """
+        Format class for ls_link parsed messages (openbmp.parsed.ls_link)
+
+        Schema Version: 1.3
+    """
+
+    def __init__(self, version, data):
+        """
+        Handle the message by parsing it and storing the data in memory.
+
+        :param version: Schema version of the message.
+        :param data: Data to parse.
+        """
+
+        super(LsLink, self).__init__()
+        self.spec_version = version
+
+        if version >= float(1.3):
```

Comment:
We can change this to support adding to the list/array instead of duplicating.   Moody's changes also take this into account..